<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>设备管理</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>主功能区</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>扫描设备</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>连接设备</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>断开设备</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QPushButton" name="btn_scanDevices">
                   <property name="text">
                    <string>扫描设备</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_connect">
                   <property name="text">
                    <string>连接设备</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_disconnect">
                   <property name="text">
                    <string>断开设备</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="maximumSize">
                    <size>
                     <width>60</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                     <kerning>true</kerning>
                    </font>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::RightToLeft</enum>
                   </property>
                   <property name="lineWidth">
                    <number>1</number>
                   </property>
                   <property name="text">
                    <string>设备列表</string>
                   </property>
                   <property name="textFormat">
                    <enum>Qt::AutoText</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="cbb_Devices"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>设备句柄</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="tb_Handle"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="0" column="2">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="btn_getInfo">
                 <property name="text">
                  <string>获取设备信息</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLineEdit" name="le_AFManufacturer"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="le_AFProduct"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="le_AFSerial"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="1" column="0" colspan="2">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QPushButton" name="btn_connectFunc">
                 <property name="text">
                  <string>注册连接成功回调函数</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="btn_unconnectFunc">
                 <property name="text">
                  <string>注销连接成功回调函数</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="btn_disconnectFunc">
                 <property name="text">
                  <string>注册断开连接成功回调函数</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="btn_undisconnectFunc">
                 <property name="text">
                  <string>注销断开连接成功回调函数</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="2">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QPushButton" name="btn_enableBusMsg">
                   <property name="text">
                    <string>启动总线信息统计</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_getBusMsg">
                   <property name="text">
                    <string>读取统计信息</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_ClearBusMsg">
                   <property name="text">
                    <string>清除统计信息</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QComboBox" name="cbb_BusMsgType">
                   <property name="mouseTracking">
                    <bool>false</bool>
                   </property>
                   <property name="editable">
                    <bool>false</bool>
                   </property>
                   <property name="currentText">
                    <string/>
                   </property>
                   <property name="currentIndex">
                    <number>-1</number>
                   </property>
                   <property name="placeholderText">
                    <string/>
                   </property>
                   <property name="duplicatesEnabled">
                    <bool>false</bool>
                   </property>
                   <property name="modelColumn">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lb_BusMsg">
                   <property name="maximumSize">
                    <size>
                     <width>20</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>0%</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>CAN API</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>报文信息</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tv_MsgData">
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
